numba_dppy/examples/kernel_private_memory.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020, 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dpctl
+import numpy as np
+from numba import float32
+
+import numba_dppy
+
+
+def private_memory():
+    """
+    This example demonstrates the usage of numba-dppy's `private.array`
+    intrinsic function. The function is used to create a static array
+    allocated on the devices private address space.
+    """
+
+    @numba_dppy.kernel
+    def private_memory_kernel(A):
+        prvt_mem = numba_dppy.private.array(shape=1, dtype=np.float32)
```

Comment:
`numba_dppy.private.array()`. Why use nested namespaces `private` and `local`? Why not `numba_dppy.private_array()` and `numba_dppy.local_array()`?